nautobot_circuit_maintenance/handle_notifications/handler.py

```diff
@@ -305,10 +304,10 @@ def process_raw_notification(logger: Job, notification: MaintenanceNotification)
 
 def get_since_reference(logger: Job) -> int:
     """Get the timestamp from the latest processed RawNotification or a reference from config `initial_days_since`."""
-    # Latest retrieved notification will limit the scope of notifications to retrieve
+    # Latest processed RawNotification will limit the scope of notifications to retrieve
     last_raw_notification = RawNotification.objects.last()
     if last_raw_notification:
-        since_reference = last_raw_notification.date.timestamp()
+        since_reference = last_raw_notification.created.timestamp()
```

Comment:
Just a note that `created` is a date, not a datetime. So there's some loss in granularity here - want to make sure we don't end up missing notifications as a result.

Reply:
The `SINCE` filter for the email search is also a `date` not a `datetime`, so we are good.